aten/src/ATen/native/cuda/TensorShape.cu

```diff
@@ -3,13 +3,19 @@
 #include <ATen/core/Tensor.h>
 #include <ATen/cuda/CUDAContext.h>
 #include <ATen/native/Resize.h>
+#include <ATen/native/TensorShape.h>
```

Comment:
nit: can you run clang-format on this file?

@albanD - Do you know how to turn it on for a file by default? IIRC I've seen the linter run clang-format on some .cu files.